pruner/window.go

```diff
@@ -6,12 +6,16 @@ import (
 
 type AvailabilityWindow time.Duration
 
+func (aw AvailabilityWindow) Duration() time.Duration {
+	return time.Duration(aw)
+}
```

Comment:
q: Is there a reason to make AvailabilityWindow a new type or more like a feeling? I think that could be an alias that wouldn't need the method

Reply:
Can address in a separate PR